app/assets/javascripts/components/shared/modal.js

```diff
@@ -24,6 +24,11 @@
         showTitle: false,
         // Title of the modal – mandatory for accessibility
         title: 'Untitled modal',
+        // Allows the modal container have scroll
+        allowScroll: false,
```

Comment:
By default, the scroll is always available so there's no reason to "allow" it. It already is.

Also, the rest of the modals didn't have any issue. We're just letting the user scrolls if they want. Moreover, disabling the scroll as you're doing makes mobile devices scroll to top, so it's more an inconvenient than a solution.